[master < 801-formatting-email-username-when-creating-an-user] Update docs with new info (#836)
memgraph/docs: docs/how-to-guides/manage-user-privileges.md

CREATE USER alice IDENTIFIED BY '0042';
```

> If you are creating a new username that is an email address, you need to enclose it in backticks (``` ` ```):

Reviewer: ```suggestion
If the username is an email address, you need to enclose it in backticks (``` ` ```):
```
